renderlib.py

```diff
@@ -195,11 +195,21 @@ def events(scheduleUrl, titlemap={}):
 			# iterate events on that day in this room
 			for event in room.iter('event'):
 				# aggregate names of the persons holding this talk
+				persons = []
 				personnames = []
+				personnames2 = []
 				if event.find('persons') is not None:
 					for person in event.find('persons').iter('person'):
 						personname = re.sub( '\s+', ' ', person.text ).strip()
-						personnames.append(personname)
+						personname = personname.split(",")[0]
```

Comment:
d.h. dass Personennamen mit Komma nicht mehr komplett gerendert werden. Solche Sachen gehören eher in dne tasks()-Block in des Projektfiles rein, da kannst du mit den Daten machen was du magst :)